css/material.less

```diff
@@ -0,0 +1,8 @@
+label {
+  margin-top: 0px;
+}
+
+input[type="text"], input[type="number"]{
```

Comment:
Are we using this class or do we know we will need it? i.e. let's double check it's needed and that we are not adding noise.

Reply:
Idem